### ISSUE TITLE
Element Call: customise base URL in release builds

### ISSUE THREAD

**Issue by YamatoRyou** [element-hq/element-x-android#2897]
#### What would you like to do?
Requests allow Element X to point to self-hosted Element Call instances, just like Element Web does.

#### Why would you like to do it?
I have my own Element Call instance, but Element X doesn't seem to make video calls through it to my self-hosted Element Web instance. This causes connectivity issues on the following occasions:
1. An Android phone (Device A, running Element X, pointed to my self-hosted Matrix server) and a Windows PC (Device B, running Element Web, pointed to my self-hosted Matrix server, and the Element Web configuration file added extra content to point to my own Element Call instance);
2. If device A initiates a video call first, device B will try to enter the video room for a long time (this looks like a connection failure from the user's perspective);
3. If device B initiates the video call first, device A can connect smoothly, although sometimes it will be slower.
4. In my area, Device A is still slow even with Element Call only turned on.

#### How would you like to achieve it?
n/a

### Have you considered any alternatives?

I'm still using the Element legacy client to avoid issues with poor connectivity on voice or video calls.

### Additional context

**element.io connection is very slow in my area and even blocked sometimes.**

### Are you willing to provide a PR?

No

**Comment by jmartinesp** [comment]
In debug and nightly builds there is an option to change the Element Call base URL in developer options, but it's not exposed in release builds. I believe having the homeserver automatically configure this is something we want to build in the future, but in the meanwhile using a nightly version could be a workaround for this.

**Comment by YamatoRyou** [comment]
> In debug and nightly builds there is an option to change the Element Call base URL in developer options, but it's not exposed in release builds. I believe having the homeserver automatically configure this is something we want to build in the future, but in the meanwhile using a nightly version could be a workaround for this.

I just tried the nightly version, and according to your instructions, this problem has indeed been solved.
However, some new problems were discovered:
1. The system language currently used by device A is Simplified Chinese. At this time, the display language of Element X is also Simplified Chinese. However, when I changed the URL of the Element Call, I found that the Element Call is still displayed in English. Known methods in the browser It is to append `?lang=zh-hans` at the end of the URL of Element Call, but this method is not applicable in Element X;
2. Element X uses a dark appearance, but Element Call fails to respect this setting.
